spanner/tests/system.py

```diff
@@ -75,9 +85,6 @@ def setUpModule():
     Config.INSTANCE_CONFIG = configs[0]
     config_name = configs[0].name
 
-    def _list_instances():
-        return list(Config.CLIENT.list_instances())
-
     instances = retry(_list_instances)()
```

Comment:
-0.  I don't really see the benefit in moving this function out to module scope.